scripts/build/p-llvm.inc

```diff
@@ -132,8 +132,8 @@ build_llvm() {
        cd "${SANITIZER_LLVM_LIBCXX}"
        cmake -GNinja -DCMAKE_BUILD_TYPE=RelWithDebInfo \
         -DLLVM_USE_SANITIZER=MemoryWithOrigins \
-        "${SANITIZER_CMAKE_C_COMPILER}" \
-        "${SANITIZER_CMAKE_CXX_COMPILER}" \
+        "${SANITIZER_CMAKE_C_COMPILER[*]}" \
```

Comment:
This should be "[@]"; next line as well

Reply:
Done, I also changed the two corresponding lines below this.